src/modules/navigation/components/Navigation/Navigation.consts.tsx

```diff
@@ -224,6 +225,12 @@ const getDynamicHeaderLinks = (
 			iconName,
 			tooltip,
 		}: NavigationInfo): NavigationItem => {
+			const hasActiveVisits = activeVisits && activeVisits.length > 0;
+			const isSearchNavItem = contentPath === ROUTES.search && hasActiveVisits;
+			const searchUrl = isSearchNavItem
+				? `${contentPath}?aanbieder=${activeVisits[0].spaceSlug}`
```

Comment:
Will `activeVisits[0].spaceSlug` always be filled up? Maybe add a fallback in case it doesn't.